### PR TITLE
update to FixedEffectModels 1.9.4

### LINKED ISSUE #130: Update for StatsModels 0.7
Is it possible to update this package to support StatsModels 0.7? The main change there was a change to FunctionTerm

### PR DESCRIPTION
**Pull request by greimel**
Closes #130, partially addresses #52

### CODE REVIEW

**Comment by greimel** [comment]
If we want to support FixedEffectModels 1.9 and MixedModels we need to require julia 1.8

(all versions that support StatsModels 0.7 require julia 1.8)

https://github.com/JuliaStats/MixedModels.jl/blob/main/NEWS.md#mixedmodels-v490-release-notes

**Comment by greimel** [comment]
closes #130

File: src/util/util.jl [hunk 1]
```diff
@@ -36,8 +36,9 @@ function name(t::InteractionTerm)
...
-end
-function name(t::FixedEffectTerm)
-    string(t.x)
-end
+    if t.f === FixedEffectModels.fe
+        string(t.exorig.args[end])
+    else
+        StatsBase.coefnames(t)
+    end
+end
```
Comment (greimel) [inline src/util/util.jl:44]: Sorry to bother you @kleinschmidt and @matthieugomez 

In a regression table we want to treat `FixedEffectTerm` differently from other `FunctionTerms`. 

E.g., for the term `fe(x)` we want to get `"x"`, while for all other functions `log(x)` we want to get `"log(x)"`.

It seems now that we cannot dispatch on `FixedEffectTerm` anymore. This code works, but I was wondering if there is a better way to handle this?

Comment (kleinschmidt) [reply src/util/util.jl:44]: yeah that's basically what I'd do.

**Comment by greimel** [comment]
FixedEffectModels removed some fields (r2 and adjr2), so changed the code to use accessor functions whenever possible. That means that `adjr2` is now printed for `LinearModel` (previously it wasn't). @jmboehm are you fine with this change?

**Comment by greimel** [comment]
Merging this, because I consider

> That means that `adjr2` is now printed for `LinearModel` (previously it wasn't). 

a bug fix. (#52)

**Pull request merged**